src/main/java/io/supertokens/inmemorydb/Start.java

```diff
@@ -1749,4 +1738,13 @@ public void removeExpiredCodes()
             throw new StorageQueryException(e);
         }
     }
+
+    @Override
+    public void deleteAllDataForUser(String userId) throws StorageQueryException {
+        try {
+            TOTPQueries.deleteAllDataForUser(this, userId);
+        } catch (StorageTransactionLogicException e) {
+            throw new StorageQueryException(e);
+        }
+    }
```

Comment:
this function is probably not needed anymore - see comment in queries file

Reply:
But we need something for `deleteNonAuthRecipeUser` right?